pull-request.sh

```diff
@@ -254,7 +264,22 @@ main () {
                                 "${ASSIGNEES}" "${REVIEWERS}" "${TEAM_REVIEWERS}"
 
         fi
+    fi
+}
+
+# Run curl with default values
+curl_wrapper() {
+    printf "curl -fsSL -H 'AUTH...' %s\n" "$*" >&2
+    set +e
+    curl -fsSL -H "${AUTH_HEADER}" -H "${HEADER}" "$@"
+    return $?
+}
 
+# Print the response and, if FAIL_ON_ERROR, exit with an error
+abort_if_fail() {
+    echo "$@"
```

Comment:
Let's use printf here to be consistent.
```suggestion
    printf "$@\n"
```

Reply:
I'll change it to printf:

```
bash-5.0$ foo() { printf "$@\n"; }
bash-5.0$ foo one two three
onebash-5.0$
bash-5.0$
bash-5.0$ foo() { printf "%s\n" "$*"; }
bash-5.0$ foo one two three
one two three
bash-5.0$ foo() { printf "%s\n" "$@"; }
bash-5.0$ foo one two three
one
two
three
```